optimizer/lib/transformers/AddBlurryImagePlaceholders.js

```diff
@@ -43,11 +45,16 @@ class AddBlurryImagePlaceholders {
       if (tagName === 'amp-video' && node.attribs.poster) {
         src = node.attribs.poster;
       }
-      if (src && !this.hasPlaceholder_(node)) {
-        promises.push(this.addBlurryPlaceholder_(tree, src).then((imgChild) => {
-          node.appendChild(imgChild);
-        }));
-      }
+      if (currPlaceholderAmt < MAX_BLURRED_PLACEHOLDERS && src &&
+        !this.hasPlaceholder_(node) && this.passesHueristic_(node, src, tagName)) {
```

Comment:
s/passesHueristic/passesHeuristic

Reply:
Or how about `shouldAddBlurryPlaceholder_`?

Reply:
+1 to @choumx